vncviewer/org.tigervnc.vncviewer.metainfo.xml.in

```diff
@@ -42,4 +42,6 @@
       <image>https://raw.githubusercontent.com/TigerVNC/tigervnc/741d3edbfab65eda6f033078bc06347fe244ea6a/vncviewer/metainfo/tigervnc-connection-windows.jpg</image>
     </screenshot>
   </screenshots>
+  <developer_name>Pierre Ossman (Cendio AB) et al.</developer_name>
```

Comment:
The specification states that this field is deprecated and that `<developer>` should be used instead. With an `id` attribute as well.

Reply:
I have updated this but also kept the developer_name tag, as even in Debian Trixie, almost all meta info files still contain the developer_name tag, even if they have the new developer tag.